pmoves/tests/smoke/test_critical_path.py

```
@@ -0,0 +1,312 @@
+        try:
+            data = response.json()
+            assert "status" in data, "Response missing 'status' field"
+
+            # Check dependency connections (optional for basic health)
+            # These might be False if deps are not configured, but status should still be healthy
+            deps_ok = True
+            for dep in ["qdrant_connected", "neo4j_connected", "meilisearch_connected"]:
+                if dep in data and not data[dep]:
+                    # Dependency is reported but not connected
+                    # This is a warning, not a failure for smoke tests
+                    deps_ok = False
```

Ruff (0.14.10)

Local variable `deps_ok` is assigned to but never used

Remove assignment to unused variable `deps_ok`

(F841)